static/specification.html

```diff
@@ -144,595 +204,516 @@ <h2>
     </section>
     <section>
       <h2>
-        Extensions to the `Document` interface
+        Link type "monetization"
       </h2>
-      <pre class="idl">
-        partial interface Document {
-          readonly attribute Monetization monetization;
-        };
-      </pre>
+      <aside class="issue">
+        <p>
+          This is a "monkey patch" for the [[HTML]] specification: If accepted,
+          this will eventually be moved to HTML itself.
+        </p>
+      </aside>
+      <aside class="note" title="Multiple monetization links">
+        <p>
+          Although a single HTML document can have multiple [^link^] elements
+          with the "monetization" link type, the user agent might restrict the
+          number of concurrent [[STREAM]] connections it can simultaneously
+          support. In such a case, an "error" event will be fired at the
```

Comment:
Will the "error" event be of type "ErrorEvent" ?

Reply:
Nevermind, I guess it's not really that suitable

Reply:
Though window.onerror emits an ErrorEvent it seems

Reply:
ErrorEvent seems appropriate... let me check...

Reply:
The nice thing about ErrorEvent is it supports communicating something more than just "error" via the "error: Error" member. But I wonder if it would be that appropriate when it comes to native implementations? It seems to be for "script" errors

Reply:
> The ErrorEvent interface represents events providing information related to errors in scripts ** or in files. **
Or in (cpp) files :) ??

Reply:
Actually, no... it seems just regular errors are thrown, as ErrorEvent are for script errors.

Reply:
Pity :/